operational/fnms/schedule_fnms_reports/CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
-v1.4
-----
+# Changelog
+
+## v2.0
+
+- Added support for onPrem FlexNet Manager. (Still need to be available on the
```

Comment:
change onPrem to on-premise.  Remove "(Still need to be available on the Internet)"